--- a/tests/test_simple.py
+++ b/tests/test_simple.py
@@ -24,11 +24,11 @@
         writer.close()
 
     server = await asyncio.start_server(
-        serve_file, port=unused_tcp_port, loop=event_loop
+        serve_file, port=unused_tcp_port
     )
 
     reader, _ = await asyncio.open_connection(
-        host="localhost", port=unused_tcp_port, loop=event_loop
+        host="localhost", port=unused_tcp_port
     )
     payload = await reader.read()
 
@@ -55,11 +55,11 @@
         writer.close()
 
     server = await asyncio.start_server(
-        serve_file, port=unused_tcp_port, loop=event_loop
+        serve_file, port=unused_tcp_port
     )
 
     reader, _ = await asyncio.open_connection(
-        host="localhost", port=unused_tcp_port, loop=event_loop
+        host="localhost", port=unused_tcp_port
     )
     payload = await reader.read()
 
